kumascript/src/api/mdn.js

```diff
@@ -1,9 +1,15 @@
 /**
  * @prettier
  */
+const fs = require("fs");
 const url = require("url");
+const path = require("path");
```

Comment:
These "two" above not not needed anymore.

Reply:
Oops, thanks @peterbe!